CHANGES_NEXT_RELEASE

```diff
@@ -1,6 +1,10 @@
-- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logging into a file
-- Add: support 'true' and 'false' literals (in addition to 'TRUE' and 'FALSE') in flag-like env vars
-- Fix: crash on MHD_REQUEST_TERMINATED_CLIENT_ABORT situations (#3738)
-- Upgrade Dockerfile base image from centos7.7.1908 to centos7.9.2009
-- FIX BUG: Create normal files as 644 permissions for assure backup tools take configuration files
-
+- Fix: notification.includePayload to avoid sending payloads in custom notification (#3272)
+- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logging into a file
+- Add: support 'true' and 'false' literals (in addition to 'TRUE' and 'FALSE') in flag-like env vars
+- Upgrade Dockerfile base image from centos7.7.1908 to centos7.9.2009
+- FIX BUG: Create normal files as 644 permissions for assure backup tools take configuration files
+- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logging into a file
+- Add: support 'true' and 'false' literals (in addition to 'TRUE' and 'FALSE') in flag-like env vars
+- Fix: crash on MHD_REQUEST_TERMINATED_CLIENT_ABORT situations (#3738)
+- Upgrade Dockerfile base image from centos7.7.1908 to centos7.9.2009
+- FIX BUG: Create normal files as 644 permissions for assure backup tools take configuration files
```

Comment:
It seems a conflict in this file was not correctly solved. The file should show the current content in master + the lines corresponding to the changes in your PR.

I think that after accepting this suggestion it will be fixed:

```suggestion
- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logging into a file
- Add: support 'true' and 'false' literals (in addition to 'TRUE' and 'FALSE') in flag-like env vars
- Fix: notification.includePayload to configure sending payloads or not in custom notification (#3272)
- Fix: crash on MHD_REQUEST_TERMINATED_CLIENT_ABORT situations (#3738)
- Upgrade Dockerfile base image from centos7.7.1908 to centos7.9.2009
- FIX BUG: Create normal files as 644 permissions for assure backup tools take configuration files
```

Reply:
Fixed in 2e3fef2